tests/Mouf/Database/TDBM/TDBMDaoGeneratorTest.php

```diff
@@ -36,25 +30,35 @@
  */
 class TDBMDaoGeneratorTest extends TDBMAbsctractServiceTest {
 
-    /** @var TDBMDaoGenerator $tdbmDaoGenerator */
-    protected $tdbmDaoGenerator;
-
-    protected function setUp() {
-        $this->tdbmDaoGenerator = new TDBMDaoGenerator($this->dbConnection);
-    }
-
-    /**
-     * @depends testFake
-     */
 	public function testDaoGeneration() {
-        $daoFactoryClassName = "daoFactory";
-        $sourcedirectory = "Mouf/Database/TDBM/Dao";
-        $daonamespace = "Mouf/Database/TDBM/Dao/";
-        $beannamespace = "Mouf/Database/TDBM/Dao/Bean/";
+        $daoFactoryClassName = "DaoFactory";
+        // Put your right directory from your home
+        // Todo, find a good way to do it
+        $sourcedirectory = "projects/mouf/database.tdbm/tests/Mouf/Database/TDBM";
```

Comment:
Yes, for the moment there is no test cases for DAOs and Beans generation. For the moment, I don't have a lot of time. I will finish them after the 15th August, is it ok for you ?